jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -45,20 +47,47 @@ public Builder setLabel(String label) {
     }
 
     /**
-     * Adds an entry to the layer.
+     * Adds an entry to the layer. Only adds the single source file to the exact path in the
+     * container file system.
      *
-     * <p>The source files are specified as a list instead of a set to define the order in which
-     * they are added.
+     * <p>For example, {@code addEntry(Paths.get("myfile"), Paths.get("/path/in/container"))} would
+     * add {@code myfile} to the container to be accessed at {@code /path/in/container}.
```

Comment:
This language is still ambiguous.  I think you mean _to be accessed **as**_ (not **at**).